### PR TITLE
Updates to WorkLock documentation

### PR DESCRIPTION
**Pull request by cygnusv**

### CODE REVIEW

**Review by vzotova — APPROVED** [review]

File: docs/source/architecture/worklock.rst [hunk 1]
```diff
@@ -263,42 +263,42 @@ the maximum stake amount (30,000,000 NU).
...
+        - `whale_1` is refunded ~ 22.46 ETH
+        - `whale_2` (i.e. you) is refunded ~ 22.46 ETH + 360 ETH (from Step 3) ~ 382.46 ETH
 
     #. Based on the refunds
 
         - The bonus escrows for the whales will now be equalized:
 
-            - `whale_1` bonus ~ 6,100 ETH - 184.14 ETH ~ 5,915.86 ETH
-            - `whale_2` bonus ~ 6,300 ETH - 384.14 ETH ~ 5,915.86 ETH
+            - `whale_1` bonus ~ 6,390 ETH - 22.36 ETH ~ 6,367.54 ETH
```
Comment (derekpierre) [inline docs/source/architecture/worklock.rst:280]:
```suggestion
            - `whale_1` bonus ~ 6,390 ETH - 22.46 ETH ~ 6,367.54 ETH
```

**Review by KPrasch — APPROVED** [review]
Need a third and fourth pair of eyes on the numbers.  Including @derekpierre 's RFCs, this looks good to me.  Hopefully this will answer some common questions over the next few weeks! :cowboy_hat_face:

**Review by derekpierre — CHANGES_REQUESTED** [review]
see RFCs

**Review by derekpierre — APPROVED** [review]
🎸

**Review by mswilkison — APPROVED** [review]

**Pull request merged**